app/views/appointment_summaries/new.html.erb

```diff
@@ -7,7 +7,13 @@
 <% end %>
 
 <%= form_for @appointment_summary, url: appointment_summaries_path do |f| %>
-  <%= f.label :name %>
-  <%= f.text_field :name, class: 't-name' %>
+  <div class="l-form-row">
+    <%= f.label :name %>
+    <%= f.text_field :name, class: 't-name' %>
+  </div>
+  <div class="l-form-row">
+    <%= f.label :email_address %>
+    <%= f.text_field :email_address, class: 't-email_address' %>
```

Comment:
`t-email_address` or `t-email-address` considering we have `l-form-row` above it